README.md

```diff
@@ -18,6 +18,19 @@ https://flashpaper.io
   3. Run `docker-compose up -d` to start FlashPaper
   4. Set up a reverse-proxy in front of FlashPaper that terminates SSL/TLS
 
+#### Building an Image
+  You can build your own image using the provided Dockerfile in the `docker/` folder. There are currently two:
+  - For x86_64 (`docker/Dockerfile`)
+  - For ARM64 (`docker/arm.Dockerfile`)
+  In order to build FlashPaper, run `docker build . -t flashpaper -f docker/Dockerfile`. If you would like to build FlashPaper for a different CPU architecture, replace `docker/Dockerfile` with the appropriate Dockerfile.
```

Comment:
There should be a line break here so that "For ARM64" and the "In order to build FlashPaper" are on separate lines.